translations/de_CH/atrium-translations-de_CH-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionIterableAssertion.kt

```diff
@@ -46,7 +46,9 @@ enum class DescriptionIterableAssertion(override val value: String) : StringBase
     NEXT_ELEMENT("ein nächstes Element"),
     NO_ELEMENTS("❗❗ kann nicht eruiert werden, leeres Iterable"),
     DUPLICATE_ELEMENTS("doppelte Elemente"),
-    DUPLICATED_BY("(dupliziert von Index: %s")
+    DUPLICATED_BY("(dupliziert von Index: %s"),
+    ELEMENT_NOT_FOUND("aber es konnte kein solches Element gefunden werden"),
+    NUMBER_OF_ELEMENTS_FOUND(/* TODO translate */ "and %s such elements were found")
```

Comment:
If this is satisfactory, then more translations are needed

Reply:
`und % Elemente wurden gefunden`